src/main/scala/org/bblfsh/client/BblfshClient.scala

```diff
@@ -25,8 +27,23 @@ class BblfshClient(host: String, port: Int, maxMsgSize: Int) {
                            content = content,
                            language = BblfshClient.normalizeLanguage(lang),
                            encoding = encoding)
-    val parsed = stub.parse(req)
-    parsed
+    stub.parse(req)
+  }
+
+  def parseNative(name: String, content: String, lang: String = "",
```

Comment:
`parseNative` or `nativeParse`? We're using "native-first" naming in the SDK and protocol (I was using parse_native for Python).

Reply:
oh, my bad, fixed now.